### PR TITLE
[CP-1852] Too many user files removes default relaxation audio from t…

### PR DESCRIPTION
**Pull request by lukasmas**
…he list

Changed order in which files in Relaxation are displayed.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

File: image/system_a/data/lang/English.json [hunk 1]
```diff
@@ -715,6 +715,7 @@
   "app_bell_relaxation_looped": "looped",
   "app_bell_relaxation_loop_description": "the song will play until you turn it off",
   "app_bell_relaxation_error_message": "Unsupported media type",
+  "app_bell_relaxation_limit_error_message": "<text>File limit exceeded.<br />Not all files may display<br />correctly</text>",
```
Comment (awulkiew) [inline image/system_a/data/lang/English.json:718]:
```suggestion
  "app_bell_relaxation_limit_error_message": "<text>File limit exceeded.<br />Not all files may be displayed<br />correctly</text>",
```

File: image/system_a/data/lang/Deutsch.json [hunk 2]
```diff
@@ -666,6 +666,7 @@
   "app_bell_relaxation_looped": "geschlungen",
   "app_bell_relaxation_loop_description": "der Titel wird abgespielt, bis Sie ihn ausschalten",
   "app_bell_relaxation_error_message": "Nicht unterstütztes Dateiformat",
+  "app_bell_relaxation_limit_error_message": "<text>File limit exceeded.<br />Not all files may display<br />correctly</text>",
```
Comment (awulkiew) [inline image/system_a/data/lang/Deutsch.json:669]: We have to translate this before merging.

Comment (lukasmas) [reply image/system_a/data/lang/Deutsch.json:669]: I know, these are placeholder, and I am waiting for translations.

File: image/system_a/data/lang/English.json (hunk 1 above)
Comment (lukasmas) [reply image/system_a/data/lang/English.json:718]: This text copied from the design, and for some online tools it looks grammatically correct

Comment (lukasmas) [reply image/system_a/data/lang/English.json]: Done.

**Review by dawidwojtas-mudita — APPROVED** [review]

**Review by awulkiew — APPROVED** [review]

File: image/system_a/data/lang/Deutsch.json (hunk 2 above)
Comment (lukasmas) [reply image/system_a/data/lang/Deutsch.json]: Translations are updated

**Pull request merged**